packages/api/lambdas/bootstrap.js

```diff
@@ -16,13 +16,17 @@ const { bootstrapElasticSearch } = require('@cumulus/es-client/bootstrap');
  * @param {Object} event - AWS Lambda event input
  * @returns {Promise<Object>} a Terraform Lambda invocation response
  */
-const handler = async ({ elasticsearchHostname }) => {
+const handler = async ({ elasticsearchHostname, removeAliasConflict, testContext = {} }) => {
```

Comment:
This is the guts of the change -- 

The lambda now passes in the new optional removeAliasConflict parameter, and will *not* just swallow errors.  Given nothing is reading that return value, this seems like it makes more sense (and fails the deployment if this lambda fails).